verify-build-and-generatelists

```diff
@@ -2,16 +2,32 @@
 
 PROJECT=$1
 LOG_DIR="/var/log/openSUSE-release-tools/${PROJECT}"
+IBS_PRODUCT_PREFIX="SUSE:SLFO:Products:"
+GITEA_PRODUCTS_URL="https://src.suse.de/products/"
 [ ! -d "${LOG_DIR}" ] && mkdir ${LOG_DIR}
 
+
+ibs_to_gitea_product() {
+    printf '%s' "${GITEA_PRODUCTS_URL}$(echo "${PROJECT}" | sed "s/${IBS_PRODUCT_PREFIX}//" | sed -E 's/:/\#/g')"
+}
+
+
+pkglistgen_extra_args() {
+    if echo "${PROJECT}" | grep -Eq "${IBS_PRODUCT_PREFIX}.*" ; then
```

Comment:
technically if you want to check for a prefix you need to anchor the regex, like 
```suggestion
    if echo "${PROJECT}" | grep -Eq "^${IBS_PRODUCT_PREFIX}.*" ; then
```

Reply:
I think you could also do something like this which is imho a bit nicer to read: 

```bash

case "$PROJECT" in
    "${IBS_PRODUCT_PREFIX}"*)
        printf '%s' "--engine product_composer --git-url $(ibs_to_gitea_product)"
        ;;
     *)
       printf ''
        ;;
esac

```

Reply:
Even nicer:

```
if [[ $PROJECT == "${IBS_PRODUCT_PREFIX}"* ]]
```

Reply:
You can change the shebang to `#!/bin/bash` and use bash specific stuff. The host has bash already

Reply:
@Vogtinator I should have addressed your comment, thank you.

@dirkmueller with `=~` from Bash I was able to match explicitly the prefix with an anchor as you suggested.

I didn't change the `if then else` to `case` since there is only one branch for now and I didn't see a real improvement in readability . What about switching once/if we have more than one conditional branch? Is this a blocker in your opinion?

Thank you both for your comments